addons/test_discuss_full/tests/test_performance.py

```diff
@@ -25,7 +25,7 @@ class TestDiscussFullPerformance(HttpCase, MailCommon):
     #       - fetch res_users_settings (_format_settings)
     #       - search res_users_settings_volumes (_format_settings)
     #       - search res_lang_res_users_settings_rel (_format_settings)
-    _query_count_init_store = 11
+    _query_count_init_store = 17
```

Comment:
re-check those, they look like too big increase (should not be changed probably, as the user fields should already be prefetched)

Reply:
I didn't achieve to get those numbers down :sweat: I tried adding `prefetching=True` on the `signature` field but it didn't help.

It's possible that it's because I run the tests locally and I may not have the same configuration/modules as the runbot: When I run the test locally without my changes, I also get 17 queries there. I will undo those changes and see the query count the runbot gets.

Reply:
Ok, I undo the changes and the query counts and the tests pass. I guess that because I didn't have the same configuration/modules as the runbot :smiley: